[2022-01-11 16:39:47,655] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: stocks_dag.sql_connect_ manual__2022-01-11T16:39:46.130587+00:00 [queued]>
[2022-01-11 16:39:47,669] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: stocks_dag.sql_connect_ manual__2022-01-11T16:39:46.130587+00:00 [queued]>
[2022-01-11 16:39:47,670] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 16:39:47,671] {taskinstance.py:1239} INFO - Starting attempt 1 of 4
[2022-01-11 16:39:47,671] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 16:39:47,686] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): sql_connect_> on 2022-01-11 16:39:46.130587+00:00
[2022-01-11 16:39:47,699] {standard_task_runner.py:52} INFO - Started process 414 to run task
[2022-01-11 16:39:47,724] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'stocks_dag', 'sql_connect_', 'manual__2022-01-11T16:39:46.130587+00:00', '--job-id', '217', '--raw', '--subdir', 'DAGS_FOLDER/stocks_dag.py', '--cfg-path', '/tmp/tmpk7q6q2ha', '--error-file', '/tmp/tmpxjlp58jo']
[2022-01-11 16:39:47,726] {standard_task_runner.py:77} INFO - Job 217: Subtask sql_connect_
[2022-01-11 16:39:47,807] {logging_mixin.py:109} INFO - Running <TaskInstance: stocks_dag.sql_connect_ manual__2022-01-11T16:39:46.130587+00:00 [running]> on host 05378dffc570
[2022-01-11 16:39:47,882] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=mcgaritym@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=stocks_dag
AIRFLOW_CTX_TASK_ID=sql_connect_
AIRFLOW_CTX_EXECUTION_DATE=2022-01-11T16:39:46.130587+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-11T16:39:46.130587+00:00
[2022-01-11 16:39:47,937] {python.py:175} INFO - Done. Returned value was: Stocks Database Created
[2022-01-11 16:39:47,990] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=stocks_dag, task_id=sql_connect_, execution_date=20220111T163946, start_date=20220111T163947, end_date=20220111T163947
[2022-01-11 16:39:48,047] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-11 16:39:48,096] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
